src/styles/main.css

```diff
@@ -1,21 +1,255 @@
+.description-tabs {
+  margin-bottom: 2rem;
+  width: 20rem;
+}
+
+.description-tab {
+  float: left;
+  width: 45%;
+  text-align: center;
+  color: #ffffff;
+  font-size: 1rem;
+}
+
+.description-tab:hover {
+  color: #1a4583;
+}
+
+.description-selected-tab {
+  float: left;
+  width: 45%;
+  border-bottom: solid;
+  text-align: center;
+  color: #1a4583;
+}
+
+.description-main-row {
+  background-color: #ffffff;
+  border-radius: 0.6rem;
+  padding: .1rem;
+}
+
+.description-value {
+  font-weight: bold;
+  float: left;
+  height: 2.5rem;
+  width: 58%;
+  display: flex;
+}
+
+.description-download-value {
+  float: left;
+  height: 2.5rem;
+  width: 100%;
+  display: flex;
+  background-color: #ffffff;
+  margin-bottom: 1.5rem;
+  padding: .3rem;
+}
+
+.file-icon {
+  color: #107eec;
+  margin-right: 1rem;
+  margin-left: 1rem;
+}
+
+.form-header-container {
+  height: 10.68rem;
+  max-width: 100%;
+  margin-left: -0.5rem;
+  margin-top: 5.125rem;
+  border-color: #ffffff;
+  background-color: #27aae1;
+}
+.description-title {
+  font-size: 1.363rem;
+  color: #ffffff;
+  margin-top: 1rem;
+  margin-bottom: 2rem;
+  margin-left: 1%;
+}
 
+.description-main-title {
+  font-size: 1.363rem;
+  color: #ffffff;
+  margin-top: 1rem;
+  margin-left: 1%;
+}
+
+.description-file-link {
+  color: #27aae1;
+  font-size: 1.1rem;
+}
+
+.description-back {
+  color: #ffffff;
+  font-size: 1.1rem;
+}
+
+.description-back:hover {
+  color: #1a4583;
+}
+
+.description-container{
+  margin: 1%;
+  max-width: 49%;
+  min-width:48%;
+  height: 1rem;
+  float: left;
+  width: 48%;
+}
+
+.description-row {
+  width: 100%;
+  height: 7rem;
+  margin-left: 4%;
+}
+
+.description-form {
+  border-radius: 0.838rem;
+  padding-right: 1%;
+}
 
 .rcorners1 {
-  border-radius: 0.9375rem;
+  border-radius: 0.5375rem;
   border: 0.0625rem solid #ccc;
 }
-.button {
-  display: flex;
+.publish-button {
```

Comment:
move common properties from publish-button and cancel-button to a separate class